pages/portfolio.html

```diff
@@ -5,8 +5,44 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
```

Comment:
Make sure to update the title for each page. This is the text that shows up in the tab of the browser.